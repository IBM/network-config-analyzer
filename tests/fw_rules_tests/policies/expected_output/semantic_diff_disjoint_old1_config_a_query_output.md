|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|semantic_diff, config1: old1, config2: config_a, key: Lost connections between removed peers||||||
||[default,kube-system,vendor-system]|[*]|[default,kube-system,vendor-system]|[*]|All connections|
|semantic_diff, config1: old1, config2: config_a, key: Lost connections between removed peers and ipBlocks||||||
|||0.0.0.0/0|[default,kube-system,vendor-system]|[*]|All connections|
|||::/0|[default,kube-system,vendor-system]|[*]|All connections|
||[default,kube-system,vendor-system]|[*]||0.0.0.0/0|All connections|
||[default,kube-system,vendor-system]|[*]||::/0|All connections|
|semantic_diff, config1: old1, config2: config_a, key: New connections between added peers||||||
||[default]|[*]|[default]|[app=app-1]|All connections|
||[default]|[app in (app-1,app-2)]|[default]|[*]|All connections|
|semantic_diff, config1: old1, config2: config_a, key: New connections between added peers and ipBlocks||||||
|||0.0.0.0/0|[default]|[app=app-1]|All connections|
|||::/0|[default]|[app=app-1]|All connections|
||[default]|[*]||0.0.0.0/0|All connections|
||[default]|[*]||::/0|All connections|

