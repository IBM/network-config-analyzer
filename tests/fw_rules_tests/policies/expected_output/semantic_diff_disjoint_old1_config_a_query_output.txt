old1 and config_a are not semantically equivalent.

Lost connections between removed peers (based on topology from config: old1) :
src_ns: [default,kube-system,vendor-system] src_pods: [*] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections

Lost connections between removed peers and ipBlocks (based on topology from config: old1) :
src: 0.0.0.0/0 dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
src_ns: [default,kube-system,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections

New connections between added peers (based on topology from config: config_a) :
src_ns: [default] src_pods: [*] dst_ns: [default] dst_pods: [app!=app-1] conn: All connections
src_ns: [default] src_pods: [app in (app-1,app-2)] dst_ns: [default] dst_pods: [*] conn: All connections

New connections between added peers and ipBlocks (based on topology from config: config_a) :
src: 0.0.0.0/0 dst_ns: [default] dst_pods: [app!=app-1] conn: All connections
src_ns: [default] src_pods: [*] dst: 0.0.0.0/0 conn: All connections